cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -558,6 +558,35 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: skipper-proxy
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.233
+            args:
+            - skipper
+            - -address=:8443
+            - -tls-cert=/etc/kubernetes/ssl/apiserver.pem
+            - -tls-key=/etc/kubernetes/ssl/apiserver-key.pem
+            - -inline-routes
+            - 'z: JWTPayloadAnyKV("iss", "https://identity.zalando.com") -> disableAccessLog() -> "https://127.0.0.1:443"; all: * -> enableAccessLog() -> "https://127.0.0.1:443";'
+            ports:
+            - containerPort: 8443
+            readinessProbe:
+              httpGet:
+                path: /metrics
+                port: 9911
```

Comment:
@szuecs Do you know of a better endpoint to use for readinessProbe?

Reply:
how about injecting a route for the readiness? This would have the semantic meaning of "routing is successfully configured ergo ready to serve".

Reply:
readinessProbe should be pass through, such that the pod is ready to serve traffic. Skipper alone might not be what we want.

Reply:
I used a new route `/kube-system/healthz` that points to the `/healthz` of the API server. I didn't want to use the real `/healthz` because it needs to be forwarded to the insecure endpoint  (readiness prober doesn't have credentials) and that could leak tokens.